subprojects/packagefiles/sdl2/find-dylib-name.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+import subprocess
+import os
+import re
+import sys
+import pathlib
+
+
+def verbose(*args):
```

Comment:
This file should have a comment explaining what it does. It is not obvious from the code.

Reply:
I added a comment at the top of the file to explain what it does and when it is used